views/layouts/sidebar/content.erb

```diff
@@ -65,7 +65,7 @@
           <% end %>
         </ul>
       </li>
-      <% if @project_data %>
+      <% if @project_data && (@project_permissions & ["Project:user", "Project:policy", "Project:billing", "Project:view"]).any? %>
```

Comment:
If user doesn't have any permission on project, I don't want to show empty "Project Details" header.
 
<img width="282" alt="Screenshot 2023-08-15 at 12 56 26" src="https://github.com/ubicloud/ubicloud/assets/993199/3bf19226-eb20-4bdf-bfca-ecaa8e9f21fb">